deploy/materialized_view_fuel.sql

```diff
@@ -0,0 +1,54 @@
+-- Deploy ggircs:materialized_view_fuel to pg
+-- requires: materialized_view_unit
+
+BEGIN;
+
+-- Fuels from Units
+-- todo: explore any other attributes for units
+create materialized view ggircs_swrs.fuel as (
+  with x as (
+    select ghgr_import_id,
+           id              as unit_id,
+           activity_id,
+           unit_name,
+           xml_hunk        as unit_xml_hunk
+    from ggircs_swrs.unit
+    order by ghgr_import_id desc
+  )
+  select row_number() over (order by x.unit_id asc) as id,
+         ghgr_import_id,
+         activity_id,
+         unit_id,
+         unit_name,
+         fuel_details.*
+
+  from x,
+       xmltable(
+           '/Unit/Fuels/Fuel'
+           passing unit_xml_hunk
+           columns
+             fuel_type varchar(1000) path './FuelType',
+             fuel_classification varchar(1000) path './FuelClassification',
+             fuel_units varchar(1000) path './FuelUnits',
```

Comment:
@wenzowski Here is another example (like in units) where the namespace is going to be ggircs_swrs.fuel.fuel_type etc. Should we leave it this way in the swrs schema because it matches the xml and change it in the public schema?

Reply:
I'm going to think more on this, but yes let's assume that's the approach. Looks good :)